test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -165,11 +170,12 @@ def check_unknown(self):
         Check for unknown test suite and test case
         """
         log.debug('check_unknown called')
-        only_ts = self.__tests_list_copy.pop(self._only_ts)
-        only_tc = self.__tests_list_copy.pop(self._only_tc)
+        tests_list_copy = copy.deepcopy(self.__tests_list_copy)
```

Comment:
are you modifying the contents of self.__tests_list_copy in this function? I don't see them getting modified, in which case you don't need the deepcopy

Reply:
tests_list_copy.pop removes items. That's why copying self.__tests_list_copy to another variable.